benchmark/fs/bench-cpSync.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const path = require('path');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bench = common.createBenchmark(main, {
+  n: [1, 100, 10_000],
+});
+
+function main({ n }) {
+  tmpdir.refresh();
+  const options = { force: true, recursive: true };
```

Comment:
`force` defaults to `true`.

```suggestion
  const options = { recursive: true };
```

Reply:
Agreed. Will merge this since this is not a blocking review, but I'll definitely update this once I have the change to improve this benchmark.